scraper.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TABLE IF NOT EXISTS section
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    datetime_creation TIMESTAMP,
+    datetime_update TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS keyword
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    datetime_creation TIMESTAMP,
+    datetime_update TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS provider
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    datetime_creation TIMESTAMP,
+    datetime_update TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS type
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    datetime_creation TIMESTAMP,
+    datetime_update TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS subject
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    datetime_creation TIMESTAMP,
+    datetime_update TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS publication
+(
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(1024),
+    url TEXT,
+    pdf_name VARCHAR(1024),
+    file_hash VARCHAR(32),
+    authors VARCHAR(1024),
```

Comment:
you know the data, I don't. Is 1024 a suitable amount longer than the maximum current value? (I tend to be paranoid and go with 2x max existing value)

Reply:
It should be enough as it succesfully ran with the fairly long titles from the NICE evidences (terrible things like "comitee reunion of the xxxxx concerning xxxx and xxx for the article about xxxx reviewed by xxxx etc.)